etc/config/RTG/rtg.cfg

```diff
@@ -330,8 +330,8 @@ volcanoes {
 
     # 1/x chance that a volcano will generate in a biome that has volcanoes enabled.
     # 1 = Always generate if possible; 2 = 50% chance; 4 = 25% chance
-    #  [range: 1 ~ 2147483647, default: 6]
-    I:"Volcano Chance"=6
```

Comment:
The description here and in some other places is wrong, we ought to regenerate the configs.